src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDrop_TestPage.xaml.cs

```diff
@@ -0,0 +1,393 @@
+						break;
+					}
+				case nameof(UIElement.DragLeave):
+					{
+						eventDetails.EventBackground = this.DragLeaveBrush;
+						break;
+					}
+				case nameof(UIElement.DragOver):
+					{
+						eventDetails.EventBackground = this.DragOverBrush;
+						break;
+					}
+				case nameof(UIElement.Drop):
+					{
+						eventDetails.EventBackground = this.DropBrush;
+						break;
+					}
+				case nameof(UIElement.DropCompleted):
+					{
+						eventDetails.EventBackground = this.DropCompletedBrush;
+						break;
+					}
+			}
+
+			this.AddToDragDropEvents(eventDetails);
+
+			return;
+		}
+
+		/// <summary>
+		/// Displays the contents of the given <see cref="global::Windows.UI.Xaml.DragEventArgs"/> after a drop occurs.
+		/// </summary>
+		/// <param name="args">The drop event arguments to display.</param>
+		private async void ShowDropDetails(global::Windows.UI.Xaml.DragEventArgs args)
+		{
+			string title = "Last Drop Details";
+			string position = string.Empty;
+			string details = string.Empty;
+
+			this.DropDetailsTextBlock.Visibility = Visibility.Visible;
+			this.DropDetailsImage.Visibility = Visibility.Collapsed;
+
+			// Only one data format can be displayed at once.
+			// Therefore, the order/priority here is used to determine which should be displayed.
+			// It should be ordered most specific to least generally speaking.
+			if (args.DataView.Contains(StandardDataFormats.ApplicationLink))
+			{
+				title += " (ApplicationLink)";
+
+				var appLink = await args.DataView.GetApplicationLinkAsync();
+				details = appLink.ToString();
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Bitmap))
+			{
+				title += " (Bitmap)";
+
+				var stream = await args.DataView.GetBitmapAsync();
+
+				var bitmapImage = new BitmapImage();
+				bitmapImage.SetSource(await stream.OpenReadAsync());
+
+				this.DropDetailsImage.Source = bitmapImage;
+				this.DropDetailsImage.Visibility = Visibility.Visible;
+
+				this.DropDetailsTextBlock.Visibility = Visibility.Collapsed;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Html))
+			{
+				title += " (Html)";
+
+				var html = await args.DataView.GetHtmlFormatAsync();
+				details = html;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Rtf))
+			{
+				title += " (Rtf)";
+
+				var rtf = await args.DataView.GetRtfAsync();
+				details = rtf;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.StorageItems))
+			{
+				title += " (StorageItems)";
+
+				var items = await args.DataView.GetStorageItemsAsync();
+				if (items.Count > 0)
+				{
+					// Only use the first item
+					foreach (IStorageItem item in items)
+					{
+						var file = item as StorageFile;
+
+						if (file != null)
+						{
+							details += "Name: " + file.Name + Environment.NewLine;
+							details += "DisplayName: " + file.DisplayName + Environment.NewLine;
+							//details += "DisplayType: " + file.DisplayType + Environment.NewLine;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6334034)